{"uid":"88460c9856f42387","name":"Bank account transfer for native user","fullName":"test.test_costumer_login.TestCostumerLogin#test_costumer_login_pege","historyId":"81073f16d66b974b0c2542bf9ff983f0","time":{"start":1692706570707,"stop":1692706577209,"duration":6502},"status":"broken","statusMessage":"selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\n  (Session info: headless chrome=115.0.5790.170); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x55a0835a6613 <unknown>\n#1 0x55a0832d2537 <unknown>\n#2 0x55a0832d6ade <unknown>\n#3 0x55a0832d8340 <unknown>\n#4 0x55a0832d83fc <unknown>\n#5 0x55a0833169d6 <unknown>\n#6 0x55a083314c99 <unknown>\n#7 0x55a083312f68 <unknown>\n#8 0x55a0833120ad <unknown>\n#9 0x55a0833071a9 <unknown>\n#10 0x55a083331a32 <unknown>\n#11 0x55a083306bc6 <unknown>\n#12 0x55a083331bfe <unknown>\n#13 0x55a083349d09 <unknown>\n#14 0x55a083331803 <unknown>\n#15 0x55a08330568b <unknown>\n#16 0x55a08330642e <unknown>\n#17 0x55a083567c28 <unknown>\n#18 0x55a08356baf7 <unknown>\n#19 0x55a08357601c <unknown>\n#20 0x55a08356c723 <unknown>\n#21 0x55a08353b457 <unknown>\n#22 0x55a083590628 <unknown>\n#23 0x55a0835907b7 <unknown>\n#24 0x55a08359f483 <unknown>\n#25 0x7f6967351b43 <unknown>","statusTrace":"self = <test.test_costumer_login.TestCostumerLogin object at 0x7f2aa8bc5cc0>\n\n    @allure.title(\"Bank account transfer for native user\")\n    @allure.severity(\"NORMAL\")\n    @pytest.mark.smoke\n    def test_costumer_login_pege(self):\n        self.costumer_login_page.open()\n        self.costumer_login_page.is_open_browser()\n        self.costumer_login_page.navigate_to_costumer_login()\n        self.costumer_login_page.replenishment_deposit()\n>       self.costumer_login_page.withdrawn_sum()\n\ntest/test_costumer_login.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nfeature_pages/costumer_login_page.py:41: in withdrawn_sum\n    self.select_element_is_visibility(self.locator.AMOUNT_INPUT).click()\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click\n    self._execute(Command.CLICK_ELEMENT)\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute\n    return self._parent.execute(command, params)\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f2aa8b6df30>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"stale element reference\",\"message\":\"stale element reference: stale elemen...\\\\n#23 0x55a0835907b7 \\\\u003Cunknown>\\\\n#24 0x55a08359f483 \\\\u003Cunknown>\\\\n#25 0x7f6967351b43 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\nE         (Session info: headless chrome=115.0.5790.170); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nE       Stacktrace:\nE       #0 0x55a0835a6613 <unknown>\nE       #1 0x55a0832d2537 <unknown>\nE       #2 0x55a0832d6ade <unknown>\nE       #3 0x55a0832d8340 <unknown>\nE       #4 0x55a0832d83fc <unknown>\nE       #5 0x55a0833169d6 <unknown>\nE       #6 0x55a083314c99 <unknown>\nE       #7 0x55a083312f68 <unknown>\nE       #8 0x55a0833120ad <unknown>\nE       #9 0x55a0833071a9 <unknown>\nE       #10 0x55a083331a32 <unknown>\nE       #11 0x55a083306bc6 <unknown>\nE       #12 0x55a083331bfe <unknown>\nE       #13 0x55a083349d09 <unknown>\nE       #14 0x55a083331803 <unknown>\nE       #15 0x55a08330568b <unknown>\nE       #16 0x55a08330642e <unknown>\nE       #17 0x55a083567c28 <unknown>\nE       #18 0x55a08356baf7 <unknown>\nE       #19 0x55a08357601c <unknown>\nE       #20 0x55a08356c723 <unknown>\nE       #21 0x55a08353b457 <unknown>\nE       #22 0x55a083590628 <unknown>\nE       #23 0x55a0835907b7 <unknown>\nE       #24 0x55a08359f483 <unknown>\nE       #25 0x7f6967351b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: StaleElementReferenceException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1692706562996,"stop":1692706563021,"duration":25},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0},{"name":"driver","time":{"start":1692706563022,"stop":1692706570706,"duration":7684},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0},{"name":"setup","time":{"start":1692706570706,"stop":1692706570706,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0}],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\n  (Session info: headless chrome=115.0.5790.170); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x55a0835a6613 <unknown>\n#1 0x55a0832d2537 <unknown>\n#2 0x55a0832d6ade <unknown>\n#3 0x55a0832d8340 <unknown>\n#4 0x55a0832d83fc <unknown>\n#5 0x55a0833169d6 <unknown>\n#6 0x55a083314c99 <unknown>\n#7 0x55a083312f68 <unknown>\n#8 0x55a0833120ad <unknown>\n#9 0x55a0833071a9 <unknown>\n#10 0x55a083331a32 <unknown>\n#11 0x55a083306bc6 <unknown>\n#12 0x55a083331bfe <unknown>\n#13 0x55a083349d09 <unknown>\n#14 0x55a083331803 <unknown>\n#15 0x55a08330568b <unknown>\n#16 0x55a08330642e <unknown>\n#17 0x55a083567c28 <unknown>\n#18 0x55a08356baf7 <unknown>\n#19 0x55a08357601c <unknown>\n#20 0x55a08356c723 <unknown>\n#21 0x55a08353b457 <unknown>\n#22 0x55a083590628 <unknown>\n#23 0x55a0835907b7 <unknown>\n#24 0x55a08359f483 <unknown>\n#25 0x7f6967351b43 <unknown>","statusTrace":"self = <test.test_costumer_login.TestCostumerLogin object at 0x7f2aa8bc5cc0>\n\n    @allure.title(\"Bank account transfer for native user\")\n    @allure.severity(\"NORMAL\")\n    @pytest.mark.smoke\n    def test_costumer_login_pege(self):\n        self.costumer_login_page.open()\n        self.costumer_login_page.is_open_browser()\n        self.costumer_login_page.navigate_to_costumer_login()\n        self.costumer_login_page.replenishment_deposit()\n>       self.costumer_login_page.withdrawn_sum()\n\ntest/test_costumer_login.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nfeature_pages/costumer_login_page.py:41: in withdrawn_sum\n    self.select_element_is_visibility(self.locator.AMOUNT_INPUT).click()\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click\n    self._execute(Command.CLICK_ELEMENT)\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute\n    return self._parent.execute(command, params)\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f2aa8b6df30>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"stale element reference\",\"message\":\"stale element reference: stale elemen...\\\\n#23 0x55a0835907b7 \\\\u003Cunknown>\\\\n#24 0x55a08359f483 \\\\u003Cunknown>\\\\n#25 0x7f6967351b43 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\nE         (Session info: headless chrome=115.0.5790.170); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nE       Stacktrace:\nE       #0 0x55a0835a6613 <unknown>\nE       #1 0x55a0832d2537 <unknown>\nE       #2 0x55a0832d6ade <unknown>\nE       #3 0x55a0832d8340 <unknown>\nE       #4 0x55a0832d83fc <unknown>\nE       #5 0x55a0833169d6 <unknown>\nE       #6 0x55a083314c99 <unknown>\nE       #7 0x55a083312f68 <unknown>\nE       #8 0x55a0833120ad <unknown>\nE       #9 0x55a0833071a9 <unknown>\nE       #10 0x55a083331a32 <unknown>\nE       #11 0x55a083306bc6 <unknown>\nE       #12 0x55a083331bfe <unknown>\nE       #13 0x55a083349d09 <unknown>\nE       #14 0x55a083331803 <unknown>\nE       #15 0x55a08330568b <unknown>\nE       #16 0x55a08330642e <unknown>\nE       #17 0x55a083567c28 <unknown>\nE       #18 0x55a08356baf7 <unknown>\nE       #19 0x55a08357601c <unknown>\nE       #20 0x55a08356c723 <unknown>\nE       #21 0x55a08353b457 <unknown>\nE       #22 0x55a083590628 <unknown>\nE       #23 0x55a0835907b7 <unknown>\nE       #24 0x55a08359f483 <unknown>\nE       #25 0x7f6967351b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: StaleElementReferenceException","steps":[{"name":"Open  browser and navigate to https://www.globalsqa.com/angularJs-protractor/BankingProject/#/login","time":{"start":1692706570707,"stop":1692706571933,"duration":1226},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0},{"name":"Login your account and checking welcome title","time":{"start":1692706571940,"stop":1692706575520,"duration":3580},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0},{"name":"Replenishment deposit and checking balance","time":{"start":1692706575521,"stop":1692706577017,"duration":1496},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0},{"name":"Transfer sum","time":{"start":1692706577017,"stop":1692706577208,"duration":191},"status":"broken","statusMessage":"selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\n  (Session info: headless chrome=115.0.5790.170); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x55a0835a6613 <unknown>\n#1 0x55a0832d2537 <unknown>\n#2 0x55a0832d6ade <unknown>\n#3 0x55a0832d8340 <unknown>\n#4 0x55a0832d83fc <unknown>\n#5 0x55a0833169d6 <unknown>\n#6 0x55a083314c99 <unknown>\n#7 0x55a083312f68 <unknown>\n#8 0x55a0833120ad <unknown>\n#9 0x55a0833071a9 <unknown>\n#10 0x55a083331a32 <unknown>\n#11 0x55a083306bc6 <unknown>\n#12 0x55a083331bfe <unknown>\n#13 0x55a083349d09 <unknown>\n#14 0x55a083331803 <unknown>\n#15 0x55a08330568b <unknown>\n#16 0x55a08330642e <unknown>\n#17 0x55a083567c28 <unknown>\n#18 0x55a08356baf7 <unknown>\n#19 0x55a08357601c <unknown>\n#20 0x55a08356c723 <unknown>\n#21 0x55a08353b457 <unknown>\n#22 0x55a083590628 <unknown>\n#23 0x55a0835907b7 <unknown>\n#24 0x55a08359f483 <unknown>\n#25 0x7f6967351b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/example_test_e2e_bank_app/example_test_e2e_bank_app/feature_pages/costumer_login_page.py\", line 41, in withdrawn_sum\n    self.select_element_is_visibility(self.locator.AMOUNT_INPUT).click()\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"hasContent":true,"stepsCount":0,"attachmentsCount":0}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"hasContent":true,"stepsCount":4,"attachmentsCount":0},"afterStages":[{"name":"driver::0","time":{"start":1692706577938,"stop":1692706577994,"duration":56},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0}],"labels":[{"name":"feature","value":"Login page"},{"name":"severity","value":"NORMAL"},{"name":"tag","value":"smoke"},{"name":"parentSuite","value":"test"},{"name":"suite","value":"test_costumer_login"},{"name":"subSuite","value":"TestCostumerLogin"},{"name":"host","value":"fv-az572-169"},{"name":"thread","value":"2301-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.test_costumer_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["smoke"]},"source":"88460c9856f42387.json","parameterValues":[]}